runtime/rust-api/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "rust-api"
```

Comment:
`near-contract` is the API, and `rust-api` is the API + tests?
We should make an API crate that doesn't depend on near runtime, here is my suggestion:
- `near-contract` standalone crate
- `near-stubs` with implementations of binding.rs functions (not sure if we can make all methods make sense but just storage write/read will be useful) to include in a contract's tests
- `contract-examples` with a contract + unit tests (`near-contract` in dependencies, `near-stubs` in dev-dependencies)
- integration tests using examples compiled to wasm and executor + MyExt interfaces

Reply:
`near-contract` is a procedural macro crate. There is not `rust-api` crate, but we need to put integration tests somewhere therefore we create bodyless crate called `rust-api` that encapsulates all `rust-api`-related stuff and integration tests. `rust-api` dev-depends on runtime because that's what integration tests are using. I can extract stuff the way you suggested.